samples/boards/stm32/pm_lptim/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _Power-Management-LP-Timer:
+
+Power Management LP Timer
+#########################
```

Comment:
Since this sample is named after the LPTIM, it would be useful to explain why this particular timer is required to test the low-power modes.